homeassistant/components/google/__init__.py

```diff
@@ -1,14 +1,4 @@
-"""
-Support for Google - Calendar Event Devices.
-
-For more details about this platform, please refer to the documentation at
-https://home-assistant.io/components/google/
-
-NOTE TO OTHER DEVELOPERS: IF YOU ADD MORE SCOPES TO THE OAUTH THAN JUST
```

Comment:
This NOTE will be lost

Reply:
Was moved to the documentation -> https://github.com/home-assistant/home-assistant.io/commit/7e0d3ab4ab41a26535a6198e09cd6598b75bf10c